OCP 4.15 Release Notes for P/Z (#70664)
openshift/openshift-docs: release_notes/ocp-4-15-release-notes.adoc

* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on {ibm-power-title}]
* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on {ibm-power-title} in a restricted network]

[IMPORTANT]
====
Compute nodes must run {op-system-first}
====

[discrete]
==== {ibm-power-title} notable enhancements

The {ibm-power-title} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.

This release introduces support for the following features on {ibm-power-title}:

* Agent-Based Installer
* cert-manager Operator for Red Hat OpenShift
* {ibm-power-title} Virtual Server Block CSI Driver Operator
* Installer-provisioned Infrastructure Enablement for {ibm-power-title} Virtual Server
* nx-gzip for Power (Hardware Acceleration)
* openshift-install to support various SMT levels on Power (Hardware Acceleration)

[discrete]
=== {ibm-power-title}, {ibm-z-title}, and {ibm-linuxone-title} support matrix

> Starting in {product-title} 4.15, Extended Update Support (EUS) is extended to the {ibm-power-title} and the {ibm-z-title} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].

Reviewer: ```suggestion
Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibm-power-title} and the {ibm-z-title} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
```

[id="ocp-4-15-ibm-z"]
=== {ibm-z-title} and {ibm-linuxone-title}
With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. The installation can be performed with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see the following documentation:

* xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Installing a cluster with on {ibm-z-title} and {ibm-linuxone-title}]

[IMPORTANT]
====
Compute nodes must run {op-system-first}
====

[discrete]
==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements

The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.

This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:

* Agent-based Installer

> * {cert-manager-operator}

Reviewer: ```suggestion
* {cert-manager-operator}
* `s390x` control plane with `x86_64` multi-architecture compute nodes
```

[discrete]
==== Installing a cluster with an LPAR on {ibm-z-title} and {ibm-linuxone-title}

{product-title} now supports user-provisioned installation of {product-title} 4.15 with a logical partition (LPAR) on {ibm-z-title} and {ibm-linuxone-title}

For installation instructions, see the following documentation:

> * xref:../installing/installing_ibm_z/installing-ibm-z-kvm.html#installing-rhcos-using-ibm-secure-execution_installing-ibm-z-kvm[Installing a cluster with an LPAR on {ibm-z-name} and {ibm-linuxone-name}]

Reviewer: ```suggestion
* xref:../installing/installing_ibm_z/installing-ibm-z-lpar.adoc#installing-ibm-z-lpar[Installing a cluster in an LPAR on {ibm-z-name} and {ibm-linuxone-name}]
* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-lpar.adoc#installing-restricted-networks-ibm-z-lpar[Installing a cluster in an LPAR on {ibm-z-name} and {ibm-linuxone-name} in a restricted network]

```

> //update above with the correct link when the PR is merged.

Reviewer: ```suggestion

```

[id="ocp-4-15-ibm-power"]
=== {ibm-power-title}

{ibm-power-name} is now compatible with {product-title} {product-version}. For installation instructions, see the following documentation:

* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on {ibm-power-name}]
* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on {ibm-power-name} in a restricted network]

[IMPORTANT]
====
Compute nodes must run {op-system-first}
====

[discrete]
==== {ibm-power-title} notable enhancements

The {ibm-power-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.

This release introduces support for the following features on {ibm-power-name}:

* Agent-based Installer
* {cert-manager-operator}
* {ibm-power-name} Virtual Server Block CSI Driver Operator
* Installer-provisioned Infrastructure Enablement for {ibm-power-name} Virtual Server

> * Multi-architecture Control Plane with support of Intel and Power workers

Reviewer 2: ```suggestion
* Multi-architecture Power Control Plane with support of Intel and Power workers
```

Author: Updated the PR. Thanks @prb112

[id="ocp-4-15-ibm-z"]
=== {ibm-z-title} and {ibm-linuxone-title}

> With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. The installation can be performed with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see the following documentation:

Reviewer 3: ```suggestion
With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. You can perform the installation with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see the following documentation:
```

* xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Installing a cluster with on {ibm-z-title} and {ibm-linuxone-title}]

[IMPORTANT]
====

> Compute nodes must run {op-system-first}

Reviewer 3: ```suggestion
Compute nodes must run {op-system-first}.
```

====

[discrete]
==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements

The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.

This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:

* Agent-based Installer
* {cert-manager-operator}
* `s390x` control plane with `x86_64` multi-architecture compute nodes

[discrete]
==== Installing a cluster with an LPAR on {ibm-z-title} and {ibm-linuxone-title}

> {product-title} now supports user-provisioned installation of {product-title} 4.15 with a logical partition (LPAR) on {ibm-z-title} and {ibm-linuxone-title}

Reviewer 3: ```suggestion
{product-title} now supports user-provisioned installation of {product-title} 4.15 with a logical partition (LPAR) on {ibm-z-title} and {ibm-linuxone-title}.
```

For installation instructions, see the following documentation:

* xref:../installing/installing_ibm_z/installing-ibm-z-lpar.adoc#installing-ibm-z-lpar[Installing a cluster in an LPAR on {ibm-z-name} and {ibm-linuxone-name}]

* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-lpar.adoc#installing-restricted-networks-ibm-z-lpar[Installing a cluster in an LPAR on {ibm-z-name} and {ibm-linuxone-name} in a restricted network]

[id="ocp-4-15-ibm-power"]
=== {ibm-power-title}

{ibm-power-name} is now compatible with {product-title} {product-version}. For installation instructions, see the following documentation:

* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on {ibm-power-name}]
* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on {ibm-power-name} in a restricted network]

[IMPORTANT]
====

> Compute nodes must run {op-system-first}

Reviewer 3: ```suggestion
Compute nodes must run {op-system-first}.
```

====

[discrete]
==== {ibm-power-title} notable enhancements

The {ibm-power-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.

This release introduces support for the following features on {ibm-power-name}:

* Agent-based Installer
* {cert-manager-operator}
* {ibm-power-name} Virtual Server Block CSI Driver Operator
* Installer-provisioned Infrastructure Enablement for {ibm-power-name} Virtual Server
* Multi-architecture Power Control Plane with support of Intel and {ibm-power-name} workers
* nx-gzip for Power10 (Hardware Acceleration)

> * openshift-install to support various SMT levels on {ibm-power-name} (Hardware Acceleration)

Author: @prb112 request you to review above component names once again. The update is I have added {ibm-power-name}.

Reviewer 2: Looks good.

Reviewer 2: /lgtm

> * openshift-install to support various SMT levels on {ibm-power-name} (Hardware Acceleration)

Reviewer 4: Markup `openshift-install` and add something like utility/command/binary/tool after:
```suggestion
* The `openshift-install` utility to support various SMT levels on {ibm-power-name} (Hardware Acceleration)
```
I couldn't find a definitive answer for what to call it, but those were some options from around the repo.

|Supported
|Supported

|Hosted control planes (Technology Preview)
|Supported
|Supported

|IBM Secure Execution
|Unsupported
|Supported

> |{ibm-power-server-name} Block CSI Driver Operator

Reviewer: @JoeAldinger If you look at the table headers you can see that column 2 is IBM Power, column 3 is IBM Z. 
The two features are supported on IBM Power but not on IBM Z and the plan is to move from TP to GA with 4.15.

> |{ibm-power-server-name} Block CSI Driver Operator

Reviewer 5: Sorry, my confusion.

|====
|Feature |{ibm-power-name} |{ibm-z-name} and {ibm-linuxone-name}

> |cert-manager Operator for Red Hat OpenShift

Reviewer: ```suggestion
|{cert-manager-operator}
```
